app/controllers/tasks_controller.rb

```diff
@@ -61,54 +57,29 @@ def destroy
 
   private
 
+  def set_search
+    search = params[:search]
+    @created_before_days = search[:created_before_days] if search.is_a?(ActionController::Parameters)
+    @option = params[:option]
+    @dropdown = params[:dropdownWindowActive]
```

Comment:
I was wondering whether we should use some better describing name here as well. What about "advancedFilterActive", "advancedFilterVisible", "advancedFilterUsed", ...?

This would separate the variable naming from the implementation detail.